src/checkstatus/get_dbstatus_function.py

```diff
@@ -5,10 +5,13 @@
 
 def lambda_get_dbinstance_status(event, context):
     """Method to obtain status of a RDS instance post actions such as snapshot creation, rename, restore or delete"""
-    region = os.environ['Region']
+    taskname = event['output']['taskname']
+    if taskname == constants.COPY_SNAPSHOT:
```

Comment:
Hi @namitad 

I am including the ternary operator as I agree it might be more elegant that way. Though I see redundancy of adding additional structure for a separate method for a one-liner. I will kindly leave the line without creating a new method. Please let me know if there is some additional benefit that I didn't notice.

Reply:
@despot presently this method is polluted with the logic of client creation based on region when the responsibility of the method should be to get the status. Moving it to a separate method will promote separation of concerns

Reply:
@namitad I agree creation logic can be viewed as separation of concern, though it doesn't seem extracting a method for a one liner is a good idea. I will do it anyway since you are insisting.